components/ide/jetbrains/backend-plugin/src/main/resources-latest/META-INF/extensions.xml

```diff
@@ -7,7 +7,5 @@
     <extensions defaultExtensionNs="com.intellij">
         <projectService serviceImplementation="io.gitpod.jetbrains.remote.latest.GitpodTerminalService" client="guest" preload="true"/>
```

Comment:
This line should also be moved to `components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml`, like you did for `GitpodMetricControlProvider` and `GitpodMetricProvider`.

Reply:
Done!